docs/assets/docs.css

```diff
@@ -0,0 +1,328 @@
+/*
+    Taken from angular-ui/bootstrap
+    See https://github.com/angular-ui/bootstrap/blob/master/misc/demo/assets/demo.css
+*/
+
+body {
+    opacity: 1;
```

Comment:
Spacing is a bit inconsistent in this file - I see some 4 space indentations, and some 2 in places too. I'd recommend 2, but it's up to you.